### ISSUE TITLE
Unable to authorize privite Github repositories 

### ISSUE THREAD

**Issue by rsalmani** [SAP/credential-digger#260]
After using private github token, unable to scan the repositories. Receiving an error like "Repository does not exist or provided git token is invalid"

**Comment by SlimTrabelsi** [comment]
Is the token that you generated has sufficient access rights to read/write on Github ?

**Comment by rsalmani** [comment]
> Is the token that you generated has sufficient access rights to read/write on Github ?

Yes. I have given all write and read permission to token.

**Comment by rsalmani** [comment]
Yes. it's working now after removing username. can I get know the format for uploading local downloaded repos? It's not accepting the the directory path.
![image](https://user-images.githubusercontent.com/118891192/223192224-e53f8f6c-5064-4626-8196-e169eeb0303b.png)

**Comment by marcorosa** [comment]
Hi, are you using a docker container? If so, please note that you have access to the container file system and not to the host one. So if you don't have that folder INSIDE the container, you cannot scan it.

Workaround: if you use kali, you can install the the tool and you can then use it via CLI, or even run the UI via `python ui/server.py`
